material_3_demo/lib/component_screen.dart

```diff
@@ -37,9 +39,28 @@ class ComponentScreen extends StatelessWidget {
   }
 }
 
+class LayoutSetting extends StatelessWidget {
```

Comment:
Minor nit, this name doesn't tell me much about what it does. Perhaps `CenteredContstrainedWidth`, or `CenteredMaxWidth` or something like that.

Also, perhaps this alignment and constrained width could just be applied to the parent container instead of each child?

Reply:
Removed the class and applied the Alignment and width to the ListView(parent). Thanks!